daemon/dns/dns_resolvconf_file.go

```diff
@@ -13,6 +13,10 @@ import (
 const (
 	// resolvconfFilePath defines path to resolv.conf file for DNS
 	resolvconfFilePath = "/etc/resolv.conf"
+	// resolvconfFileMark mark resolv.conf file content to recorgnize
+	resolvconfFileMark = "# Generated by NordVPN"
+	// resolvconfFileContent is simple dns settings to restore as a last option
+	resolvconfFileContent = "#restored\nnameserver 1.1.1.1"
```

Comment:
Sure, let's discuss this

Reply:
We certainly should not set the DNS to external service without user consent. I think that the best solution in such case would be to set it to `nameserver 127.0.0.53`, which is local resolver address (for example from systemd). And, to handle the case when it might not be present/non working, resolve our API requests internally with our own DNS server. This will allow users to connect to NordVPN even when their system DNS resolution is not working.

Reply:
Now 3 options are tried:
1. check if 127.0.0.53:53 is active 
2. find default gateway
3. use 1.1.1.1 (should not happen)